test/block-service-test.js

```diff
@@ -21,196 +24,109 @@ module.exports = (repo) => {
 
     describe('offline', () => {
       it('store and get a block', (done) => {
-        const b = new Block('A random data block')
-        b.key((err, key) => {
-          expect(err).to.not.exist
-          const cid = new CID(key)
+        const data = new Buffer('A random data block')
+        multihashing(data, 'sha2-256', (err, hash) => {
```

Comment:
For the sake of our users, it is better to call the thing that comes out of multihashing a `multihash` because that is what it is.